lib/cusolver/wrappers.jl

```diff
@@ -256,9 +256,9 @@ end
 
 using LinearAlgebra
 using LinearAlgebra: BlasInt, checksquare
-using LinearAlgebra.LAPACK: chkargsok
+using LinearAlgebra.LAPACK: chkargsok, chklapackerror
```

Comment:
unused?

Reply:
Yes, I initially wanted to use that, but then decided not to. I will remove it.

Reply:
Now, it's used in the error checking the info code as done in `LinearAlgebra.LAPACK.potrs!`.